kernel/src/expressions/scalars.rs

```diff
@@ -290,14 +290,14 @@ impl Scalar {
         Ok(Self::Decimal(dval))
     }
 
-    /// Constructs a Scalar timestamp with no timezone from an `i64` millisecond since unix epoch
-    pub(crate) fn timestamp_ntz_from_millis(millis: i64) -> DeltaResult<Self> {
+    /// Constructs a Scalar timestamp from an `i64` millisecond since unix epoch
+    pub(crate) fn timestamp_from_millis(millis: i64) -> DeltaResult<Self> {
```

Comment:
maybe just remind ourselves that the timestamps are always UTC?
```suggestion
    /// Constructs a Scalar timestamp (in UTC) from an `i64` millisecond since unix epoch
    pub(crate) fn timestamp_from_millis(millis: i64) -> DeltaResult<Self> {
```

Reply:
If we think this is a big footgun, we _could_ rename the type as `TimestampUtc` to match `TimestampNtz`?

Reply:
I actually like that a lot for clarity. I suppose the major tradeoff is currently I think our enum matches 1:1 with the names of delta's types and this would be a departure from that..